src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -69,6 +69,7 @@ export default ajaxCaller(class LaunchAnalysisModal extends Component {
       [_.isArray(entities), () => _.uniq(entities).length],
       () => _.size(entities)
     )
+    const wrappableOnPeriods = _.flatMap(char => char === '.' ? [wbr(), '.'] : char)
```

Comment:
TOL: This might potentially create a very large array, which I'd imagine creates extra work during rendering. Perhaps we could do something like use `split` to keep the non-period substrings intact to limit that.

Reply:
What do you think of this?